pkg/investigations/etcddatabasequotalowspace/etcddatabasequotalowspace.go

```diff
@@ -68,12 +69,64 @@ func (i *Investigation) Run(rb investigation.ResourceBuilder) (investigation.Inv
 		snapshotResult.Namespace,
 		float64(snapshotResult.SnapshotSize)/(1024*1024),
 		snapshotResult.SnapshotPath)
-	r.Notes.AppendAutomation("Snapshot saved to node filesystem at: %s", snapshotResult.SnapshotPath)
-	r.Notes.AppendAutomation("Node name saved for Phase 2 job scheduling: %s", snapshotResult.NodeName)
-	r.Notes.AppendAutomation("Phase 2 will analyze this snapshot using Kubernetes Job with octosql-etcd image and hostPath mount")
 
 	logging.Infof("etcd snapshot taken successfully from pod %s on node %s", snapshotResult.PodName, snapshotResult.NodeName)
 
+	timestamp := extractTimestampFromPath(snapshotResult.SnapshotPath)
+
+	job, err := createAnalysisJob(ctx, r.K8sClient, snapshotResult.NodeName, snapshotResult.SnapshotPath, timestamp)
+	if err != nil {
+		r.Notes.AppendWarning("Failed to create analysis job: %v", err)
+		logging.Errorf("failed to create analysis job: %v", err)
+		handleSnapshotCleanup(ctx, r.K8sClient, r.Notes, snapshotResult)
+		return result, r.PdClient.EscalateIncidentWithNote(r.Notes.String())
+	}
+
+	r.Notes.AppendAutomation("Created analysis job: %s (will auto-delete after 10 minutes)", job.Name)
+
+	err = waitForJobCompletion(ctx, r.K8sClient, job.Name, 5*time.Minute)
+	if err != nil {
+		r.Notes.AppendWarning("Analysis job failed or timed out: %v", err)
+		logging.Errorf("analysis job failed: %v", err)
+		handleSnapshotCleanup(ctx, r.K8sClient, r.Notes, snapshotResult)
+		return result, r.PdClient.EscalateIncidentWithNote(r.Notes.String())
+	}
+
+	r.Notes.AppendSuccess("Analysis job completed successfully")
+
+	logs, err := getJobLogs(ctx, r.K8sClient, job.Name)
+	if err != nil {
+		r.Notes.AppendWarning("Failed to retrieve analysis results: %v", err)
+		logging.Errorf("failed to get job logs: %v", err)
+		handleSnapshotCleanup(ctx, r.K8sClient, r.Notes, snapshotResult)
+		return result, r.PdClient.EscalateIncidentWithNote(r.Notes.String())
+	}
+
+	analysisResult, err := parseAnalysisOutput(logs)
+	if err != nil {
+		r.Notes.AppendWarning("Failed to parse analysis results: %v", err)
+		logging.Errorf("failed to parse analysis output: %v", err)
+		return result, r.PdClient.EscalateIncidentWithNote(r.Notes.String())
+	}
+
+	formattedResults := formatAnalysisResults(analysisResult)
+	r.Notes.AppendSuccess("%s", formattedResults)
```

Comment:
I think we should just use the cluster report and not put the results in notes, since they could be pretty large